internal/db/migrations/005_reservations.sql

```diff
@@ -24,7 +24,7 @@ CREATE TABLE aws_reservation_details
 (
   reservation_id BIGINT NOT NULL,
   provider INTEGER NOT NULL DEFAULT provider_type_aws(),
-  pubkey_id BIGINT NOT NULL REFERENCES pubkeys(id),
+  pubkey_id BIGINT NOT NULL REFERENCES pubkeys(id) ON DELETE CASCADE,
   source_id TEXT NOT NULL,
   image_id TEXT NOT NULL,
   aws_reservation_id TEXT,
```

Comment:
We cannot do this. This will never be executed on stage anymore.

Reply:
Unfortunately you need to ALTER TABLE DROP CASCADE CREATE CASCADE, I think this all can be done in a single statement.

Reply:
I wonder why the migration check did not error out.

Reply:
Ah found it, fixed it hopefully: https://github.com/RHEnVision/provisioning-backend/pull/242

After you rebase this PR on top of this, it should fail because you touched existing SQL files...

Reply:
There should now be a new migration, and it works for me. I would be happy if you checked that. In the PR, there is no editing of an old migration, so hopefully, the tests will pass.